mne/stats/tests/test_cluster_level.py

```diff
@@ -84,6 +84,18 @@ def test_cache_dir():
             del os.environ['MNE_MEMMAP_MIN_SIZE']
 
 
+def test_perumation_large_n_samples():
```

Comment:
perumation -> permutation

Reply:
wow incorrect order and missing character